wavelength_router/wavelength_router.proto

```diff
@@ -101,3 +127,60 @@ message AdjustPSDResponse {
 
 message CancelAdjustPSDResponse {}
 
+message AdjustSpectrumRequest {
+  // Absolute path to the /components/component list member in the OpenConfig
+  // model corresponding to the port to perform the adjustment for.
+  // (e.g., /components/component[name=amp-1-1])
```

Comment:
wouldn't this be a port on the ROADM (not AMP?) in the case where they are combined, are the ports named for amplifier?

Reply:
The important part is to indicate directionality to the system as these adjustments are made on a per direction basis. Some vendors use the amp port as a convenient way, but I agree, it logically makes more sense in the example for this to be a waverouter port (it also aligns well with the name of the proto :)